content/getting-started/windows.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Windows"
+date: 2019-04-15T11:41:45+02:00
+draft: false
+weight: 3
+---
+
+This page has information on how to install and use TinyGo on Windows. The only currently supported option is to use the Docker image.
```

Comment:
I think you can also give WSL as an option. It may not be supported, but it is very likely to work (after all, most of Ubuntu runs on it) and is probably easier for users that already have WSL but not Docker.

For example:

> You can also try installing TinyGo inside WSL by following the documentation for Linux [link]. However, flashing microcontrollers may be difficult from within WSL.

Reply:
Link added.